convert.go

```diff
@@ -219,3 +245,14 @@ func (c *converter) convertTemplateFor(expr *hclsyntax.ForExpr) (string, error)
 func (c *converter) wrapExpr(expr hclsyntax.Expression) string {
 	return "${" + c.rangeSource(expr.Range()) + "}"
 }
+
+func (c *converter) convertFunctionCall(expr hclsyntax.Expression) (interface{}, error) {
+	specCtx := &hcl.EvalContext{
+		Functions: specFuncs,
+	}
+	value, valueDiags := expr.Value(specCtx)
+	if valueDiags.HasErrors() {
+		return nil, fmt.Errorf(valueDiags.Error())
```

Comment:
This should fall back to the existing behavior of wrapping the expression.

Reply:
Yeah it's not that important.